common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2934,6 +2935,14 @@ object CelebornConf extends Logging {
       .checkValues(Set(PartitionSplitMode.SOFT.name, PartitionSplitMode.HARD.name))
       .createWithDefault(PartitionSplitMode.SOFT.name)
 
+  val SHUFFLE_COMPRESSION_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.shuffle.compression.enabled")
```

Comment:
seems we can use `celeborn.client.shuffle.compression.codec=none` to represent that?

Reply:
@pan3793 done.